### ISSUE TITLE
How to get both dimensional scores and embedding in one run?

### ISSUE THREAD

**Issue by acmlxg** [audeering/w2v2-how-to#27]
Hi, thank you for sharing the great job, I want to get dimensional scores and embedding in one run, how should I do?

**Comment by frankenjoe** [comment]
That's actually the default behavior if you don't set `outputs`:

```python
import numpy as np

import audonnx


model = audonnx.load('../model')

sampling_rate = 16000
signal = np.random.normal(size=sampling_rate).astype(np.float32)
y = model(signal, sampling_rate)
print(y)
```
```
{'hidden_states': array([[-0.00717198,  0.00625039, -0.00744212, ...,  0.00660673,
         0.00962825,  0.00282657]], dtype=float32), 'logits': array([[0.6801778 , 0.65449774, 0.50805616]], dtype=float32)}
```

or:

```python
import audinterface

interface = audinterface.Feature(
    model.labels(),
    process_func=model,
    process_func_args={
        'concat': True,
    }
)
df = interface.process_signal(signal, sampling_rate)
print(df)
```
```
                        hidden_states-0  hidden_states-1  ...  dominance   valence
start  end                                                ...                     
0 days 0 days 00:00:01        -0.007172          0.00625  ...   0.654498  0.508056

[1 rows x 1027 columns]
```

**Comment by acmlxg** [comment]
> That's actually the default behavior if you don't set `outputs`:
> 
> ```python
> import numpy as np
> 
> import audonnx
> 
> 
> model = audonnx.load('../model')
> 
> sampling_rate = 16000
> signal = np.random.normal(size=sampling_rate).astype(np.float32)
> y = model(signal, sampling_rate)
> print(y)
> ```
> 
> ```
> {'hidden_states': array([[-0.00717198,  0.00625039, -0.00744212, ...,  0.00660673,
>          0.00962825,  0.00282657]], dtype=float32), 'logits': array([[0.6801778 , 0.65449774, 0.50805616]], dtype=float32)}
> ```
> 
> or:
> 
> ```python
> import audinterface
> 
> interface = audinterface.Feature(
>     model.labels(),
>     process_func=model,
>     process_func_args={
>         'concat': True,
>     }
> )
> df = interface.process_signal(signal, sampling_rate)
> print(df)
> ```
> 
> ```
>                         hidden_states-0  hidden_states-1  ...  dominance   valence
> start  end                                                ...                     
> 0 days 0 days 00:00:01        -0.007172          0.00625  ...   0.654498  0.508056
> 
> [1 rows x 1027 columns]
> ```

Thank you, it helps me a lot! 
BTW, what's the range of `singal` if we use `y = model(signal, sampling_rate)`? Actually, when I first use this code repo, I'm confused at which audio load function should I use (librosa / scipy.wavfile, etc.)

**Comment by acmlxg** [comment]
> > That's actually the default behavior if you don't set `outputs`:
> > ```python
> > import numpy as np
> > 
> > import audonnx
> > 
> > 
> > model = audonnx.load('../model')
> > 
> > sampling_rate = 16000
> > signal = np.random.normal(size=sampling_rate).astype(np.float32)
> > y = model(signal, sampling_rate)
> > print(y)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ```
> > {'hidden_states': array([[-0.00717198,  0.00625039, -0.00744212, ...,  0.00660673,
> >          0.00962825,  0.00282657]], dtype=float32), 'logits': array([[0.6801778 , 0.65449774, 0.50805616]], dtype=float32)}
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > or:
> > ```python
> > import audinterface
> > 
> > interface = audinterface.Feature(
> >     model.labels(),
> >     process_func=model,
> >     process_func_args={
> >         'concat': True,
> >     }
> > )
> > df = interface.process_signal(signal, sampling_rate)
> > print(df)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ```
> >                         hidden_states-0  hidden_states-1  ...  dominance   valence
> > start  end                                                ...                     
> > 0 days 0 days 00:00:01        -0.007172          0.00625  ...   0.654498  0.508056
> > 
> > [1 rows x 1027 columns]
> > ```
> 
> Thank you, it helps me a lot! BTW, what's the range of `singal` if we use `y = model(signal, sampling_rate)`? Actually, when I first use this code repo, I'm confused at which audio load function should I use (librosa / scipy.wavfile, etc.)

It seems that the input signal is loaded by `soundfile` and then normalize to 0-mean and 1-variance using `( x - mean ) / std`.
